contrib/message-logging/message-logging-parser.py

```diff
@@ -0,0 +1,176 @@
+#!/usr/bin/env python3
+# Copyright (c) 2020 The Bitcoin Core developers
+# Distributed under the MIT software license, see the accompanying
+# file COPYING or http://www.opensource.org/licenses/mit-license.php.
+"""Parse message log binary files.  To be used in conjunction with -logmessages."""
+
+import argparse
+import os
+import shutil
+import sys
+from io import BytesIO
+import json
+from pathlib import Path
+from typing import Any, List, Optional
+
+sys.path.append(os.path.join(os.path.dirname(__file__), '../../test/functional'))
+
+from test_framework.messages import ser_uint256     # noqa: E402
+from test_framework.mininode import MESSAGEMAP      # noqa: E402
```

Comment:
All `test_framework.mininode` need to be replaced by `test_framework.p2p` as of https://github.com/bitcoin/bitcoin/pull/19760

Reply:
Thanks!